packages/ddd/src/utils/create-angular-component.ts

```diff
@@ -0,0 +1,25 @@
+import { Tree } from '@nrwl/devkit';
+import { wrapAngularDevkitSchematic } from '@nrwl/devkit/ngcli-adapter';
+
+export interface createAngularComponentOptions {
+  name: string;
+  project: string;
+  style: string;
+  export: boolean;
+  flat: boolean;
+  prefix: string;
+}
+
+export const createAngularComponent = async (
+  tree: Tree,
+  options: createAngularComponentOptions
+): Promise<void> => {
+  const libraryGenerator = wrapAngularDevkitSchematic(
```

Comment:
I disagree about it.
Nx recommends using their libraries generators.
https://nx.dev/l/r/generators/workspace-generators

Reply:
disagree about what? I wrote that what you did here is valid because its like exec, you dont import the schematic and it doesn't have to be installed in this project, so that is a good thing.
btw what happens if you dont have the package installed and you use it? does it throw a formatted error?